app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :post
+
+  def update_comments_counter
```

Comment:
[ OPTIONAL ] It's a suggestion that you can use Active Record Call back `after_save` to call the `#update_comments_counter` method every time a new post has been saved to database, and make the `update_comments_counter` method private.